app/assets/src/components/views/projects/ProjectsView.jsx

```diff
@@ -0,0 +1,120 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Table } from "~/components/visualizations/table";
+import { merge, pick } from "lodash";
+import GlobeIcon from "~ui/icons/GlobeIcon";
+import LockIcon from "~ui/icons/LockIcon";
+import moment from "moment";
+import cs from "./projects_view.scss";
+import cx from "classnames";
+
+class ProjectsView extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.columns = [
+      {
+        dataKey: "public_access",
+        width: 30,
+        label: "",
+        cellRenderer: this.renderAccess
+      },
+      {
+        dataKey: "details",
+        label: "Project",
+        flexGrow: 1,
+        className: cs.detailsCell,
+        cellRenderer: this.renderProjectDetails,
+        headerClassName: cs.detailsHeader,
+        sortFunction: p => p.name
+      },
+      {
+        dataKey: "hosts",
+        width: 200,
+        disableSort: true,
+        cellRenderer: this.renderList
+      },
+      {
+        dataKey: "tissues",
+        width: 200,
+        disableSort: true,
+        cellRenderer: this.renderList
+      },
+      { dataKey: "number_of_samples", width: 140, label: "No. Of Samples" }
```

Comment:
nit: maybe "Number of Samples" or "No. of samples"

Reply:
This is design specification. Will bring it up with Jenn since this will not be the final version... but given that is a short column I would prefer the reduced width.